### ISSUE TITLE
Pyinstaller that targets Fedora/Centos/RHL and other rpm supporting distros

### ISSUE THREAD

**Issue by George3d6** [mindsdb/mindsdb_server#106]
Start looking into this once #103 is done

**Comment by George3d6** [comment]
In the current paradigm (with the very tiny pyinstaller generate file + a custom install script) I think this should be  almost the same as #103 but still, keeping it as a separate issue.
